src/plugins/graphql.js

```diff
@@ -172,36 +217,46 @@ function normalizeArgs (args) {
   }
 }
 
-function createOperationSpan (tracer, config, operation, source) {
+function createOperationSpan (tracer, config, operation, source, variableValues, startTime) {
   const type = operation.operation
   const name = operation.name && operation.name.value
-
   const parentScope = tracer.scopeManager().active()
+  const tags = {
+    'service.name': getService(tracer, config),
+    'resource.name': [type, name].filter(val => val).join(' '),
+    'graphql.document': source
+  }
+  if (variableValues && config.filterVariables) {
+    tags['graphql.variables'] = JSON.stringify(config.filterVariables(variableValues))
```

Comment:
Should this be a single tag or multiple tags? (similar to another PR that you made with the HTTP headers)

Reply:
I like that, just like the GraphQL document, you can copy-paste these directly into a debugging session.

Reply:
I wonder if there is a use case for individual tags though (i.e. when using Trace Search)

Reply:
Ohh wait, I’m still a bit of a newb with the UI. Yeah, being able to filter traces within a named operation group would be great indeed! I’ll make the change 👍